eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/jobs/VariantAnnotConfigurationBatchTest.java

```diff
@@ -0,0 +1,230 @@
+            cnt++;
+            VariantAnnotation annot = converter.convertToDataModelType((DBObject)cursor.next().get("annot"));
+            TestCase.assertTrue(annot.getConsequenceTypes() != null);
+            consequenceTypeCount += annot.getConsequenceTypes().size();
+        }
+
+        assertTrue(cnt>0);
+        assertEquals(1, consequenceTypeCount);
+    }
+
+    /**
+     * This simulate VEP prediction
+     *
+     * @throws IOException
+     */
+    private void makeVepAnnotationTmpGzipFile() throws IOException {
+        String vepAnnotation =
+                "20_60343_G/A\t20:63351\tG\tENSG00000178591\tENST00000608838\tTranscript\tupstream_gene_variant\t-\t-\t-\t-\t-\trs181305519\tDISTANCE=4540;STRAND=1;SYMBOL=DEFB125;SYMBOL_SOURCE=HGNC;HGNC_ID=18105;BIOTYPE=processed_transcript;GMAF=G:0.0005;AFR_MAF=G:0.0020\n";
+
+        try(FileOutputStream output = new FileOutputStream(variantJobsArgs.getPipelineOptions().getString("vepOutput"))) {
+            try(Writer writer = new OutputStreamWriter(new GZIPOutputStream(output), "UTF-8")) {
+                writer.write(vepAnnotation);
+            }
+        }
+
+    }
+
+    private String readLine(File outputFile) throws IOException {
+        if (reader == null) {
+            reader = new BufferedReader(new FileReader(outputFile));
+        }
+        return reader.readLine();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        variantJobsArgs.loadArgs();
+        vepInputFile = new File(variantJobsArgs.getPipelineOptions().getString("vepInput"));
+        converter = new DBObjectToVariantAnnotationConverter();
+
+        dbName = variantJobsArgs.getPipelineOptions().getString(VariantStorageManager.DB_NAME);
+        mongoClient = new MongoClient();
+
+        cleanDBs();
+    }
+
+    @AfterClass
+    public static void afterTests() throws UnknownHostException {
+        cleanDBs();
+    }
+
+    /**
+     * Release resources and delete the temporary output file
+     */
+    @After
+    public void tearDown() throws Exception {
+        if (reader != null) {
+            reader.close();
+        }
+
+        mongoClient.close();
+
+        vepInputFile.delete();
+        new File(variantJobsArgs.getPipelineOptions().getString("vepOutput")).delete();
+    }
+
+    private static void cleanDBs() throws UnknownHostException {
+        JobTestUtils.cleanDBs(
+                dbName
+        );
+    }
+
+    private void insertVariantsWithoutAnnotations(String databaseName, String collectionName) {
+        collection(databaseName, collectionName).insert(constructDbo(variantWithoutAnnotation));
+    }
+
+    private DBCollection collection(String databaseName, String collectionName) {
+        return mongoClient.getDB(databaseName).getCollection(collectionName);
+    }
+
+    private DBObject constructDbo(String variant) {
+        return (DBObject) JSON.parse(variant);
+    }
+
+    private final String variantWithoutAnnotation = "{\n" +
```

Comment:
Extracting to plain file into test/resources folder would simplify making any changes to test input.